src/xsar/sentinel1_dataset.py

```diff
@@ -91,13 +87,15 @@ def __init__(self, dataset_id, resolution=None,
         """`xsar.Sentinel1Meta` object"""
 
         if not isinstance(dataset_id, Sentinel1Meta):
-            xml_parser = XmlParser(
-                xpath_mappings=sentinel1_xml_mappings.xpath_mappings,
-                compounds_vars=sentinel1_xml_mappings.compounds_vars,
-                namespaces=sentinel1_xml_mappings.namespaces)
-            self.s1meta = Sentinel1Meta(dataset_id, xml_parser=xml_parser)
+            self.s1meta = BlockingActorProxy(Sentinel1Meta, dataset_id)
+            # check serializable
+            #import pickle
+            #s1meta = pickle.loads(pickle.dumps(self.s1meta))
+            #assert isinstance(tuple,s1meta.coords2ll(100, 100))
```

Comment:
Removing comment line

Reply:
This check might be turned into a test: can you add a test that check s1meta serialization ?

Reply:
I add test in my future pull request for purest

Reply:
What is variable tuple in assertion ?

Reply:
It's just a test to check that `s1meta.coords2ll(100, 100)` returns a type tuple